spec/features/viewing_and_posting_peeps_spec.rb

```diff
@@ -0,0 +1,67 @@
+feature 'Posting peeps' do
+  context 'after signing up and peeping a peep' do
+
+    before do
+      visit '/peeps'
+      click_button 'Sign up'
+
+      fill_in 'email', with: 'samm@makersacademy.com'
+      fill_in 'password', with: 'password123'
+      fill_in 'name', with: 'Sam Morgan'
+      fill_in 'username', with: 'sjmog'
+      click_button 'Submit'
+    end
+
+    before(:each) do
+      click_button 'Peep'
+
+      expect(current_path).to eq('/peeps/new')
+      fill_in 'message', with: 'First peep'
+      click_button 'Peep'
+    end
+
+    scenario 'see the message of the peep' do
+      expect(current_path).to eq('/peeps')
+      expect(page).to have_content('First peep')
+    end
+
+    scenario 'see the timestamp of the peep' do
+      expect(current_path).to eq('/peeps')
+      expect(page).to have_content('Peeped: ')
+    end
```

Comment:
Maybe not a very intuitive test to understand, since you are mentioning the timestamp, but it doesn't appear in the actual ```expect``` block.